src/auth/middlewares.go

```diff
@@ -17,6 +17,7 @@ func MandatoryAuthMiddleware(c *fiber.Ctx) error {
 		})
 	}
 	c.Locals("user", user)
+	c.Locals("user_id", user.ID)
```

Comment:
If c.Locals has user already, why do you need c.Locals to have user_id separately?

Reply:
I am facing an issue here as I am unable to type cast the local to the user model @championswimmer

Reply:
Fixed @championswimmer